ArcGISRuntimeSDKQt_CppSamples/Analysis/Geotriggers/Geotriggers.cpp

```diff
@@ -158,7 +158,8 @@ void Geotriggers::createGeotriggerMonitor(ServiceFeatureTable* serviceFeatureTab
   connect(geotriggerMonitor, &GeotriggerMonitor::geotriggerNotification, this, &Geotriggers::handleGeotriggerNotification);
 
   // Start must be explicitly called. It is called after the signal connection is defined to avoid a race condition in Qt.
```

Comment:
Review this comment. It does not sound valid anymore.

Reply:
Modified the function name in comment - startAsync.